l10n_pt_account/__openerp__.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Contabilidade PT',
+    'version': '1.122',
+    'author': 'ThinkOpen Solutions,Odoo Community Association (OCA)',
+    'category' : 'Accounting & Finance',
+    'sequence': 1,
+    'license' : 'LGPL-3',
+    'description': '''Customization of Accounting for Portugal.
```

Comment:
Move the description into a README.rst file.
Please use the template at OCA/maintainer-tools